Services/UI/OJS.Services.Ui.Business/ISubmissionsBusinessService.cs

```diff
@@ -26,6 +26,9 @@ public interface ISubmissionsBusinessService : IService
         Task<IEnumerable<SubmissionResultsServiceModel>> GetSubmissionResultsByProblem(int problemId, bool isOfficial,
             int take = 0);
 
+        Task<IEnumerable<SubmissionResultsServiceModel>> GetSubmissionResultsByProblemAndUser(int problemId, bool isOfficial, string userId,
+            int take = 0);
```

Comment:
`take = 0` as default is strange. IMO, a constant with default value of `5` or `10` is better.

@shsimeonova @mishogv @PetarNeshkov @msmner thoughts?

Reply:
There should already be a constant

Reply:
Scratch that. I created a new file with constants in the data service, since I now set the default value there.